pkgs/servers/tacacs+/default.nix

```diff
@@ -0,0 +1,21 @@
+{ stdenv, fetchurl, tcp_wrappers, flex, bison, perl }:
+
+stdenv.mkDerivation rec {
+  pname = "tacacs+";
+  version = "4.0.4.28";
+
+  src = fetchurl {
+    url = "ftp://ftp.shrubbery.net/pub/tac_plus/tacacs-F${version}.tar.gz";
+    sha256 = "17i18z3s58c8yy8jxp01q3hzz5nirs4cjxms18zzkli6ip4jszql";
+  };
+
+  buildInputs = [ tcp_wrappers flex bison perl ];
+
+  meta = with stdenv.lib; {
+    description = "TACACS+ authentication daemon";
```

Comment:
https://nixos.org/nixpkgs/manual/#sec-standard-meta-attributes
> A short (one-line) description of the package. This is shown by nix-env -q --description and also on the Nixpkgs release pages.
Don’t include a period at the end. Don’t include newline characters. Capitalise the first character. For brevity, don’t repeat the name of package — just describe what it does.
Wrong: "libpng is a library that allows you to decode PNG images."
Right: "A library for decoding PNG images"

Reply:
Fixed

Reply:
@0x4A6F specifically:
> For brevity, don’t repeat the name of package — just describe what it does.